src/applications/proxy-rewrite/redirects/crossDomainRedirects.json

```diff
@@ -709,26 +634,6 @@
     "src": "/compensation/sb2018.asp",
     "dest": "/disability/compensation-rates/birth-defect-rates/"
   },
-  {
```

Comment:
**Third chunk**
https://www.benefits.va.gov/COMPENSATION/sb2014.asp 
https://www.benefits.va.gov/COMPENSATION/sb2013.asp 
https://www.benefits.va.gov/COMPENSATION/sb2012.asp 
https://www.benefits.va.gov/COMPENSATION/sb2011.asp

**Note: DON'T do anything with these (leave as is with redirects):**
https://www.benefits.va.gov/compensation/sb2019.asp 
https://www.benefits.va.gov/COMPENSATION/sb2018.asp